datahub-web-react/src/app/entity/shared/tabs/Entity/DataFlowJobsTab.tsx

```diff
@@ -1,19 +1,43 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useBaseEntity } from '../../EntityContext';
 import { EntityType } from '../../../../../types.generated';
 import { EntityList } from './components/EntityList';
 import { useEntityRegistry } from '../../../../useEntityRegistry';
+import { useTaskPagination } from './components/TaskPaginationContext';
 
 export const DataFlowJobsTab = () => {
```

Comment:
what if instead of needing to wrap the profile in a pagination provider, we just fetch the data from this component and do the pagination in this component here?

we could always default fetch the first 10 (instead of 100 like we do now) but then paginate from this component entirely like we do in most other places.

Reply:
Done